### PR TITLE
[MON-2402] added integrations docs

### PR DESCRIPTION
**Pull request by yromanyshyn**

### CODE REVIEW

File: docs/source/_static/images/user-guide/user_interface/8.1.3_webhook_creation_form.png
Comment (yanivshani100) [inline docs/source/_static/images/user-guide/user_interface/8.1.3_webhook_creation_form.png:1]: @yromanyshyn Change the upper text to be "The URL adress where Deepchecks monitor will send a request when a anew monitor alert is triggered"

File: docs/source/user-guide/user_interface/integrations.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+
+Notifications
+=============
+
+
+Email Notifications
+-------------------
+
+To configure email notifications go to the page 'Configurations/Integrations' and click on the tab 'Notifications'.
```
Comment (yanivshani100) [inline docs/source/user-guide/user_interface/integrations.rst:13]: @yromanyshyn swap tab 'Notifications' -> to  'Notifications' tab

File: docs/source/user-guide/user_interface/integrations.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+On the shown table mark alert severity levels about which you want to be notified.
```
Comment (yanivshani100) [inline docs/source/user-guide/user_interface/integrations.rst:14]: @yromanyshyn Replace with  "On the display table mark the alert severity levels for which you would like to receive notifications on"

File: docs/source/user-guide/user_interface/integrations.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+.. image:: /_static/images/user-guide/user_interface/8.1.1_example_of_email_notification.png
+    :width: 600
+
+
+
+Slack Notifications
+-------------------
+
+To configure Slack notifications go to the page 'Configurations/Integrations' and click on the tab 'Notifications'.
```
Comment (yanivshani100) [inline docs/source/user-guide/user_interface/integrations.rst:32]: @yromanyshyn change to 'Notifications' tab

File: docs/source/user-guide/user_interface/integrations.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+
+On the table mark alert severity levels about which you want to be notified via Slack.
+
+
+.. image:: /_static/images/user-guide/user_interface/8.1.2_selection_of_alert_severity_for_slack_notifications.png
+    :width: 600
+
+
+Example of a slack message.
```
Comment (yanivshani100) [inline docs/source/user-guide/user_interface/integrations.rst:63]: @yromanyshyn add An (An example of a ...)

File: docs/source/user-guide/user_interface/integrations.rst [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+To create a webhook click on the button 'Create Webhook'.
+
+
+.. image:: /_static/images/user-guide/user_interface/8.1.3_webhook_creation.png
+    :width: 600
+
+
+Fill the form.
+
+Make sure that the specified 'Webhook URL' is reachable from the world and is ready to receive 'HTTP POST' requests.
```
Comment (yanivshani100) [inline docs/source/user-guide/user_interface/integrations.rst:87]: @yromanyshyn remove "from the world"

File: docs/source/user-guide/user_interface/integrations.rst [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+
+On the table mark alert severity levels with which the Deepchecks must send requests to the specified HTTP endpoint.
+
+
+.. image:: /_static/images/user-guide/user_interface/8.1.3_selection_of_alert_severity_for_webhook.png
+    :width: 600
+
+
+Example of an HTTP request payload.
```
Comment (yanivshani100) [inline docs/source/user-guide/user_interface/integrations.rst:102]: @yromanyshyn An example

**Review by yanivshani100 — CHANGES_REQUESTED** [review]

File: docs/source/_static/images/user-guide/user_interface/8.1.3_webhook_creation_form.png
Comment (yromanyshyn) [reply docs/source/_static/images/user-guide/user_interface/8.1.3_webhook_creation_form.png:1]: fixed

File: docs/source/user-guide/user_interface/integrations.rst (hunk 1 above)
Comment (yromanyshyn) [reply docs/source/user-guide/user_interface/integrations.rst]: fixed

File: docs/source/user-guide/user_interface/integrations.rst (hunk 6 above)
Comment (yromanyshyn) [reply docs/source/user-guide/user_interface/integrations.rst]: fixed

File: docs/source/user-guide/user_interface/integrations.rst (hunk 5 above)
Comment (yromanyshyn) [reply docs/source/user-guide/user_interface/integrations.rst]: fixed

File: docs/source/user-guide/user_interface/integrations.rst (hunk 3 above)
Comment (yromanyshyn) [reply docs/source/user-guide/user_interface/integrations.rst]: fixed

File: docs/source/user-guide/user_interface/integrations.rst (hunk 2 above)
Comment (yromanyshyn) [reply docs/source/user-guide/user_interface/integrations.rst]: fixed

**Pull request merged**